SwissTransferFeatures/UploadProgressView/UploadSuccess/UploadSuccessQRCodeView.swift

```diff
@@ -91,6 +92,21 @@ struct UploadSuccessQRCodeView: View {
             }
             .buttonStyle(.ikBorderedProminent)
         }
+        .onAppear {
+            NotificationCenter.default.addObserver(
+                forName: UIApplication.userDidTakeScreenshotNotification,
+                object: nil,
+                queue: .main
+            ) { _ in
+                Task { @MainActor in
+                    isShowingSheet = true
+                }
+            }
+        }
+        .discoveryPresenter(isPresented: $isShowingSheet) {
+            ScreenshotSheetView()
+        }
+        .matomoView(view: "ScreenshotQrCodeView")
```

Comment:
This won't work as this matomo will be triggered each time the `UploadSuccessView` is triggered